src/org/parosproxy/paros/view/View.java

```diff
@@ -168,6 +175,8 @@
     // ZAP: splash screen
     private SplashScreen splashScreen = null;
 
+    private static Map<AddOn.Status, StatusUI> statusMap = new HashMap<AddOn.Status, StatusUI>();

```

Comment:
Yup I know.

If I remove from the right I get a type safety warning.
If I also remove from the left I get a raw type warning and then in getStatusUI() I end-up with an error can't convert Object to StatusUI.